pkg/validate/validate.go

```diff
@@ -29,7 +29,7 @@ type CompareGeneratedAssetsResponse struct {
 
 // PassedValidation returns whether the response seems to indicate that the chart repositories are in sync
 func (r CompareGeneratedAssetsResponse) PassedValidation() bool {
-	return len(r.ModifiedPostRelease) == 0 && len(r.UntrackedInRelease) == 0
+	return len(r.ModifiedPostRelease) == 0 && len(r.UntrackedInRelease) == 0 && len(r.UntrackedInRelease) == 0
```

Comment:
Did you intentionally add this duplicated conditional?

Reply:
yes 😓, thank you lol

Reply:
updated